app/controllers/rake_ui/rake_task_logs_controller.rb

```diff
@@ -55,5 +53,10 @@ def rake_task_log_as_json(task)
     def rake_task_logs_as_json(tasks = [])
       tasks.map { |task| rake_task_log_as_json(task) }
     end
+
+    def klass
+      RakeUi.configuration.active_storage ? ::RakeTaskLog : RakeUi::RakeTaskLog
```

Comment:
[you said Done, above](https://github.com/suvie-eng/rake-ui/pull/2/files#r1759364439), but I still see this is not using the config.

Is the reason you're putting one in the root because you want to use the same class name, and they can't be in the same namespace?

Reply:
sorry for the confusion @NewAlexandria but `RakeTaskLog` will reside in the gem's repo so we're not assuming that the app will implement it. I forgot to push that in earlier commit but it has been pushed already now, so now you should see this class in this PR, do you want me to add more change anywhere?
I don't think even we need to add this in configuration because it is in the gem's classes we don't want the app to implement that class

Reply:
Since these are easy to confuse, I was just trying to appreciate the reason to use the same name.  I was thinking it's to make it easier for an external caller (of the gem) to have `RakeTaskLog` in any implementation.  but if there's no external implementation expectation, then yes agree this is fine